recipe/meta.yaml

```diff
@@ -1,6 +1,13 @@
 {% set version = "0.4.35" %}
 {% set name = "jaxlib" %}
 
+{% set build = 0 %}
+
+{% if cuda_compiler_version != "None" %}
+{% set build = build + 200 %}
+{% endif %}
```

Comment:
Is 200 just an arbitrary offset here to make sure the CUDA builds don't conflict with the CPU builds?

Reply:
It is somewhat arbitrary, I've seen some feedstocks use +100. I think it helps distinguish between CPU and GPU, but also favors the GPU build when the hardware and drivers are present.